docs/architecture/adr/0017-watchOS-use-swift.md

```diff
@@ -0,0 +1,57 @@
+---
+adr: "0017"
+status: In progress
```

Comment:
Should be `done`?

Reply:
Oh I didn't know if "Done" should be set there directly or wait until someone review it to pass it to done